meerk40t/gui/scenewidgets/selectionwidget.py

```diff
@@ -2458,6 +2458,8 @@ def event(
         modifiers=None,
         **kwargs,
     ):
+        if self.scene.pane.suppress_selection:
```

Comment:
This can actually be done, probably in `ToolContainer` since we're dealing with what basically amounts to switching the effective selection widget. If you see the code in ToolContainer, it already accepts that some tools may have a `selection_mode` set. NodeMoveTool for example has `select_mode = "vertex"` which lets you select the relevant nodes only.